src/change_stream.ts

```diff
@@ -46,9 +46,13 @@ const kClosed = Symbol('closed');
 const kMode = Symbol('mode');
 
 const CHANGE_STREAM_OPTIONS = ['resumeAfter', 'startAfter', 'startAtOperationTime', 'fullDocument'];
-const CURSOR_OPTIONS = ['batchSize', 'maxAwaitTimeMS', 'collation', 'readPreference'].concat(
-  CHANGE_STREAM_OPTIONS
-);
+const CURSOR_OPTIONS = [
+  'batchSize',
+  'maxAwaitTimeMS',
+  'collation',
+  'readPreference',
+  'comment'
+].concat(CHANGE_STREAM_OPTIONS);
```

Comment:
We'll have a merge conflict to resolve here when #3169 goes in. (no action, just so we catch any issue.)

Reply:
Done!